OrbitCore/LinuxTracingSession.cpp

```diff
@@ -0,0 +1,94 @@
+#include "LinuxTracingSession.h"
+
+#include <utility>
+
+void LinuxTracingSession::RecordContextSwitch(ContextSwitch&& context_switch) {
+  absl::MutexLock lock(&context_switch_buffer_mutex_);
+  context_swith_buffer_.push_back(context_switch);
```

Comment:
Since context_switch is a r-value-ref you probably want to std::move here, right?